### PR TITLE
win: support retrieving empty env variables

### PR DESCRIPTION
**Pull request by cjihrig**
Fixes: https://github.com/libuv/libuv/issues/2413

### CODE REVIEW

**Comment by cjihrig** [comment]
CI: https://ci.nodejs.org/view/libuv/job/libuv-test-commit/1524/

File: src/win/util.c [hunk 1]
```diff
@@ -1491,7 +1492,9 @@ int uv_os_getenv(const char* name, char* buffer, size_t* size) {
     if (r == ERROR_ENVVAR_NOT_FOUND)
       return UV_ENOENT;
 
-    return uv_translate_sys_error(r);
+    /* Handle defined, but empty environment variables. */
+    if (r != ERROR_SUCCESS)
```
Comment (saghul) [inline src/win/util.c]: Maybe this condition can be folded with the one above?

Comment (cjihrig) [reply src/win/util.c]: Do you mean just make this an `else if`?

Comment (saghul) [reply src/win/util.c]: I mean, if `uv_translate_sys_error` would know how to translate `ERROR_ENVVAR_NOT_FOUND` then anything `!= ERROR_SUCCESS` would just be `return uv_translate_sys_error(r);`

Comment (cjihrig) [reply src/win/util.c]: I like it 👍

**Comment by cjihrig** [comment]
CI: https://ci.nodejs.org/view/libuv/job/libuv-test-commit/1526/. None of the failures look related.

**Pull request merged**